tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -46,42 +43,43 @@ describe(__filename, () => {
 
   function render({ location, ...props } = {}) {
     const errorHandler = createStubErrorHandler();
-
-    return shallowUntilTarget(
-      <AddonMoreInfo
-        addon={props.addon || createInternalAddonWithLang(fakeAddon)}
-        errorHandler={errorHandler}
-        i18n={fakeI18n()}
-        store={store}
-        {...props}
-      />,
-      AddonMoreInfoBase,
-      {
-        shallowOptions: createContextWithFakeRouter({ location }),
-      },
-    );
+    const allProps = {
+      addon: createInternalAddonWithLang(fakeAddon),
+      errorHandler,
+      ...props,
+    };
+
+    const renderOptions = { store };
+    if (location) {
+      renderOptions.history = createMemoryHistory();
+      renderOptions.history.push(location);
+    }
+
+    return defaultRender(<AddonMoreInfo {...allProps} />, renderOptions);
   }
 
   it('renders LoadingText if no add-on is present', () => {
-    const root = render({ addon: null });
+    const { queryByClassName } = render({ addon: null });
 
     // These fields will be visible during loading since
     // they will always exist for the loaded add-on.
-    expect(root.find('.AddonMoreInfo-last-updated')).toHaveLength(1);
+    expect(screen.getByText('Last updated')).toBeInTheDocument();
 
-    expect(root.find(LoadingText)).toHaveLength(1);
+    expect(queryByClassName('LoadingText')).toHaveLength(1);
 
     // These fields will not be visible during loading
     // since they may not exist.
-    expect(root.find('.AddonMoreInfo-links')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-license')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-privacy-policy')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-version')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-version-history')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-eula')).toHaveLength(0);
+    expect(screen.queryByText('Add-on Links')).not.toBeInTheDocument();
```

Comment:
`getByText` will throw if nothing is found, which is why we need to use `queryByText` if we want to assert that something is not present in the document.